README.md

```
@@ -96,6 +99,22 @@ The files can be updated by calling
 
 The known URLs are managed in `download_urls.yml`.
 
+#### Manually maintained excerpts
+
+Some files have to be put manually in the excerpt-data folder.
+Some need to be reduced in size.
+
+- dbNSFP(a): https://usf.box.com/shared/static/2hzcx5s6p1xui7oen16xqzndfrkt8l9l
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)